File: Front/CadastroDigital-App/src/app/models/PessoaFisica.ts
import { RedeSocial } from "./RedeSocial";

export interface PessoaFisica {
  id : number;
  dataNascimento : string;
  rg : string;
  dataEmissao: string;
  orgaoExpedidorId: number | null,
  ufExpedidorId: number | null,
  naturalidadeId: number | null,
  nacionalidadeId: number | null,
  ufId: number | null,
  imagem : string;
  idUser : number;
  sexoId : number | null;
  estadoCivilId : number | null;
  redesSociais : RedeSocial[] | any;
}

File: Back/src/CadastroDigital/CadastroDigital.Domain/Entities/PessoaFisica.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CadastroDigital.Domain.Entities
{
    public class PessoaFisica : Base
    {
        public string Rg { get; set; }
        public DateTime? DataEmissao { get; set; }
        public int OrgaoExpedidorId { get; set; }
        public int UfExpedidorId { get; set; }
        public string Imagem { get; set; }
        public int IdUser { get; set; }
        public int SexoId { get; set; }
        public int EstadoCivilId { get; set; }
        public int NaturalidadeId { get; set; }

        public Identity.User User { get; set; }
        public Estado UfExpedidor { get; set; }
        public Sexo Sexo { get; set; }
        public EstadoCivil EstadoCivil { get; set; }
        public OrgaoExpedidor OrgaoExpedidor { get; set; }
        public Cidade Naturalidade { get; set; }
        public IEnumerable<RedeSocial> RedesSociais { get; set; }
        public IEnumerable<Endereco> Enderecos { get; set; }
     }
}